src/bikeracks/bikes.py

```diff
@@ -31,56 +31,45 @@ def render_file(path):
     return response
 
 
-@bikes.route('/coordinates', methods=('GET', 'POST'))
+@bikes.route('/coordinates', methods=('POST',))
 def coordinates():
-     if request.method != 'POST':
-        return "Bad Request", 400
-        
     # get the coordinates from the request
     lat = request.form.get('lat', 0, type=float)
     lng = request.form.get('lng', 0, type=float)
     address = request.form.get('address', '')
     
-    # First check if these coordinates are valid
-    if helper.validate_coordinates((lat, lng)):
-        # if valid, save in database
-        print("saving into database")
-        try:
-            # connect to database to be able to store new coordinates for temporary bikerack
-            db = get_db()
-            db.execute('INSERT INTO bikeracks (latitude, longitude, address) VALUES (?, ?, ?);', (lat, lng, address))
-            db.commit()
-            # return data for the added temporary marker
-            bike_rack = helper.get_rack_state(db, lat, lng)
-    
-            return bike_rack
-        except Exception as e:
-            print(e)
-            return "", 500
-            # TODO, narrow down exception
+    if not helper.validate_coordinates((lat, lng)):
+        return "Invalid Coordinates", 500
+
+    # if valid, save in database
+
+    # connect to database to be able to store new coordinates for temporary bikerack
+    db = get_db()
+    db.execute('INSERT INTO bikeracks (latitude, longitude, address) VALUES (?, ?, ?);', (lat, lng, address))
```

Comment:
It's considered good style to keep your lines no longer than 79 characters.

```
 db.execute(
  'INSERT INTO bikeracks (latitude, longitude, address) VALUES (?, ?, ?);',
  (lat, lng, address))
```

Reply:
Done.